src/routes/sentry-example/+page.svelte

```diff
@@ -0,0 +1,97 @@
+<!-- 
```

Comment:
TODO: Delete this before merging. We can use it in the deploy preview for testing.